src/mcts/search.cc

```diff
@@ -1435,6 +1435,12 @@ void SearchWorker::FetchSingleNodeResult(NodeToProcess* node_to_process,
                                 edge.GetMove().as_nn_index(
                                     node_to_process->probability_transform)));
   }
+  // Intermediate array to store values when processing policy.
+  // According to a lichess developer post:
```

Comment:
I think such a long comment adds more noise than usefulness, especially as it's not related to the function purpose. I'd only leave the last line of it (or combined the first and the list line if they could fit one line together).